pkgs/top-level/all-packages.nix

```diff
@@ -16221,6 +16221,8 @@ with pkgs;
 
   double-conversion = callPackage ../development/libraries/double-conversion { };
 
+  doublecmd = callPackage ../applications/misc/double-commander { };
```

Comment:
```suggestion
  doublecmd-qt = callPackage ../applications/misc/double-commander { widgetset = "qt5"; };
  doublecmd-gtk = callPackage ../applications/misc/double-commander { widgetset = "gtk2"; };
  doublecmd = doublecmd-qt;
```

Reply:
I'm still defaulting to GTK since the qt build takes about a minute to start up, no idea why.

Reply:
That's strange, I have no such problem, doublecmd-qt starts with no delay for me.